### PR TITLE
Add Breeze Theme

### PR DESCRIPTION
**Pull request by lakshayrohila**
I created this theme few days back and have used this theme only after that. It seems to resemble theme as that of Kubuntu:-
screenshot
screenshot1
screenshot2

### CODE REVIEW

**Comment by lakshayrohila** [comment]
@dimkr, please review.

**Comment by dimkr** [comment]
Looks good!

According to https://kubuntu.org/legal, "Kubuntu" is a trademark, so maybe you should consider a different name. If the icons are taken from the Breeze theme (or other copyrighted work), consider adding some README or COPYRIGHT file.

**Comment by lakshayrohila** [comment]
> Looks good!
> 
> According to https://kubuntu.org/legal, "Kubuntu" is a trademark, so maybe you should consider a different name. If the icons are taken from the Breeze theme (or other copyrighted work), consider adding some README or COPYRIGHT file.

@dimkr, Can you please recommend any other name? And where can I put the COPYRIGHT file (icons have been taken from a kde github repo. Here is the link to it-https://github.com/KDE/breeze-icons).

**Comment by dimkr** [comment]
> Can you please recommend any other name?

One of the [synonyms for "breeze"](https://www.collinsdictionary.com/dictionary/english-thesaurus/breeze) maybe?

> And where can I put the COPYRIGHT file (icons have been taken from a kde github repo).

In a directory under woof-code/rootfs-packages/jwm_config/usr/share/doc.

**Comment by lakshayrohila** [comment]
> > Can you please recommend any other name?
> 
> One of the [synonyms for "breeze"](https://www.collinsdictionary.com/dictionary/english-thesaurus/breeze) maybe?
> 
> > And where can I put the COPYRIGHT file (icons have been taken from a kde github repo).
> 
> In a directory under woof-code/rootfs-packages/jwm_config/usr/share/doc.

@dimkr, I chose **Zephyr**, 'cause I have never heard this word :sweat_smile:! Hears cool as well.

I think the only copyright file needed to be added must be https://github.com/KDE/breeze-icons/blob/master/COPYING.LIB, right?

**Comment by dimkr** [comment]
> I think the only copyright file needed to be added must be https://github.com/KDE/breeze-icons/blob/master/COPYING.LIB, right?

I think https://github.com/KDE/breeze-icons/blob/master/COPYING-ICONS, because it relates to
> The Breeze Icon Theme in icons/

**Comment by lakshayrohila** [comment]
@dimkr, I added it as `/usr/share/doc/legal/Breeze-Icons/COPYING-ICONS`. I think putting it there is okay?

**Edit:** Searching more about zephyr, I see https://github.com/zephyrproject-rtos. So, we cannot use zephyr as well. I'll first search this time.

**Review by peabee — COMMENTED** [review]
I think this should be a separate rootfs-package rather than added to jwm_config....

**Comment by dimkr** [comment]
> **Edit:** Searching more about zephyr, I see https://github.com/zephyrproject-rtos. So, we cannot use zephyr as well. I'll first search this time.

Go for Breeze then, it should be OK if the legal information is there. It's "Kubuntu" that's problematic, "Breeze" should be OK.

**Comment by lakshayrohila** [comment]
> I think this should be a separate rootfs-package rather than added to jwm_config....

It is just a JWM theme. package for this as well?

**Comment by dimkr** [comment]
> I think this should be a separate rootfs-package rather than added to jwm_config....

FYI, all other JWM themes are inside jwm_config or ptheme 🤔

**Comment by peabee** [comment]
Perhaps they should all be separated out - or at least have a small core set and "addons".......

**Review by dimkr — APPROVED** [review]
I'm fine with this PR, it's a nice addition. I don't think the one package/separate packages discussion (plus the problem of jwm_config<>ptheme cross-references) belongs in this PR.

**Comment by lakshayrohila** [comment]
> Perhaps they should all be separated out - or at least have a small core set and "addons".......

Why not keep that for another PR. By that time you could create an issue for this and others could point out which themes to be kept as the core and which ones as add_ons. One more thing, will a package be installable by a user? I think the the **Get More Themes** button might not be able to download them, just as-is. First we'll need to save the current add-on themes and then add other ones as well. Yes, you see, it grows as a bigger project, completely out of scope of a PR named **Add Breeze Theme**.

**Review by peabee — COMMENTED** [review]
OK go as is

File: woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <Font>DejaVu Sans-10</Font>
+    <Width>3</Width>
+    <Height>30</Height>
+    <Corner>4</Corner>
+    <Foreground>#ababab</Foreground>
+    <Background>#a9a9a9:#e8e8e8</Background>
+    <Outline>#303537</Outline>
+    <Active>
+      <Foreground>#ffffff</Foreground>
+      <Background>#1f2122:#1f2122</Background>
```
Comment (dimkr) [inline woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc]: You can change `#1f2122:#1f2122` to `#1f2122`

File: woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  </WindowStyle>
+
+  <TrayStyle>
+    <Font>DejaVu Sans-10</Font>
+    <Foreground>#ababab</Foreground>
+    <Background>#303537</Background>
+    <Outline>#303537</Outline>
+    <Active>
+      <Foreground>#ffffff</Foreground>
+      <Background>#1f2122:#1f2122</Background>
```
Comment (dimkr) [inline woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc]: Same here

File: woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  <PagerStyle>
+		<Outline>grey</Outline>
+		<Foreground>#e8e8e8</Foreground>
+		<Background>#a9a9a9</Background>
+		<Active>
+		<Foreground>#1f2122</Foreground>
+		<Background>#1f2122</Background>
+		</Active>
+	</PagerStyle>
```
Comment (dimkr) [inline woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc]: Indentation is messed up - mix of tabs and spaces?

File: woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	</PopupStyle>
+
+  <MenuStyle>
+    <Font>DejaVu Sans-10</Font>
+    <Foreground>#ababab</Foreground>
+    <Background>#303537</Background>
+    <Outline>#303537</Outline>
+    <Active>
+      <Foreground>#ffffff</Foreground>
+      <Background>#1f2122:#1f2122</Background>
```
Comment (dimkr) [inline woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc]: Same here

**Comment by dimkr** [comment]
@lakshayrohila https://github.com/lakshayrohila/woof-CE/pull/6

File: woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc (hunk 3 above)
Comment (lakshayrohila) [reply woof-code/rootfs-packages/jwm_config/usr/share/jwm/themes/Breeze-jwmrc]: Usage of different editors was the issue, I suppose. Thanks for fixing it!

**Pull request merged**